### ISSUE TITLE
Typescript problem with G Element in Expo 46

### ISSUE THREAD

**Issue by zagoorland** [react-native-svg/react-native-svg#1856]
Hello.
I'm using Expo 46 with React 18, and I have some problem with typescript.

This is a spinnet of used code:

```tsx
<Svg height="20" width="90%">
          <Center>
            <Icon name="plane_3" size={24} inverted />
          </Center>
          <G>
            {dashes.map((_, index) => (
              <Rect
                key={index}
                x="0"
                y="10"
                width="8"
                height="3"
                fill={colors.secondaryContrast}
                translateX={spacing * index}
              />
            ))}
          </G>
</Svg>
```

An the error I get is:
`Type '{ children: Element[]; }' has no properties in common with type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<GProps, any, any>> & Readonly<GProps>'`.

The versions I'm using:
```json
    "expo": "^46.0.0",
    "react": "18.0.0",
    "react-native": "0.69.5",
    "react-native-svg": "12.3.0",
    "@types/react": "~18.0.0",
    "@types/react-native": "~0.69.1",
```

**Comment by WoLewicki** [comment]
I believe this issue is similar to e.g. this one: https://github.com/react-native-svg/react-native-svg/issues/1798 and it should be solved since `12.4.4`. Could you check if it works correctly in this version?

**Comment by iM-GeeKy** [comment]
> I believe this issue is similar to e.g. this one: #1798 and it should be solved since `12.4.4`. Could you check if it works correctly in this version?

Upgrading to version `12.4.4` seems to resolve the issue for me. However I get the follow warning when starting Expo. 

`Some dependencies are incompatible with the installed expo version:
  react-native-svg@12.4.4 - expected version: 12.3.0
Your project may not work correctly until you install the correct versions of the packages.
Install individual packages by running npx expo install react-native-svg@12.3.0`

I've opted for the patch described in https://github.com/react-native-svg/react-native-svg/issues/1798 until this gets resolved.

**Comment by zagoorland** [comment]
> > I believe this issue is similar to e.g. this one: #1798 and it should be solved since `12.4.4`. Could you check if it works correctly in this version?
> 
> Upgrading to version `12.4.4` seems to resolve the issue for me. However I get the follow warning when starting Expo.
> 
> `Some dependencies are incompatible with the installed expo version: react-native-svg@12.4.4 - expected version: 12.3.0 Your project may not work correctly until you install the correct versions of the packages. Install individual packages by running npx expo install react-native-svg@12.3.0`
> 
> I've opted for the patch described in #1798 until this gets resolved from the Expo side. I'll open an issue on their side and link it back to this for future reference.
> 
> Here is the issue opened with the Expo team [expo/expo#19013](https://github.com/expo/expo/issues/19013)

Thanks!

**Comment by WoLewicki** [comment]
Ok so it seems like there is nothing more to be done on the side of `react-native-svg`. I will close this issue then. Feel free to comment here if something is wrong and we can always reopen it.
